module.txt

```diff
@@ -7,7 +7,12 @@
     "dependencies": [
             { "id": "AlterationEffects","minVersion": "1.0.0"},
             { "id": "BiomesAPI", "minVersion": "4.0.0"},
+<<<<<<< HEAD
+            { "id": "Health", "minVersion": "1.0.0" },
+            { "id": "Thirst", "minVersion": "1.0.0"}
+=======
             { "id": "Health", "minVersion": "1.0.0" }
+>>>>>>> e5ad36e97159dc138bd6cc4c1cfdd6f13f6f411e
```

Comment:
looks like you missed some conflicts there :wink: 
btw, it's good practise to look at this "Files changed" tab yourself once in a while

Reply:
Yes, I'm sorry, the other files turn red in color, I think I rely too much on that.
Will try to look at the "FilesChanged"  next time. Really sorry 😅